deploy/kubernetes/overlays/prow-gke-release-staging-rc/kustomization.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - ../stable
+- ./csi-driver.yaml
```

Comment:
nit: `./` shouldn't be necessary

Reply:
Thanks. I think I will leave it here because anyways we will move this to stable so it will be deleted. :)